src/js/lib/monster.ui.js

```diff
@@ -821,14 +821,27 @@ define(function(require) {
 				zIndex: 20000,
 				// Event handlers
 				close: function() {
+					var hasOtherDialogsOpen = _
+						.chain($body.find('.ui-dialog .ui-dialog-content'))
+						.map(function(element) {
+							return $(element).dialog('isOpen');
+						})
+						.some(function(isOpen) {
+							return isOpen === true;
+						})
```

Comment:
Just to be sure, the extra function is because the element may not be a dialog, and `isOpen` may not return a boolean? Because if the previous map already return booleans, `some` could be invoked without any parameter:
```suggestion
						.some()
```

Reply:
Coming back to this made me realize both operations can be solidified into a single `some` predicate, thanks for the pointer!